corpus/corpus/settings.py

```diff
@@ -57,7 +57,8 @@
     "embedathon.apps.EmbedathonConfig",
     "impulse.apps.ImpulseConfig",
     "electrika.apps.ElectrikaConfig",
-    "skyward_expedition.apps.SkywardExpeditionConfig"
+    "robotrix.apps.RobotrixConfig",
+    "skyward_expedition.apps.SkywardExpeditionConfig",
```

Comment:
Rebase with main, this should get fixed then

Reply:
did